compose/magento-2/bin/stop

```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
+set -eu
```

Comment:
Adding `set -e` here will prevent stopping of containers if mutagen daemon is not active.

Reply:
btw, I've noticed that mutagen daemon stops or kicked by the host system after some time of inactivity or when the host goes to sleep mode. Needs investigation.

Reply:
Ah, good catch…I will account for this.